hw/top_earlgrey/doc/_index.md

```diff
@@ -77,61 +86,11 @@ The block diagram of `top_earlgrey` (the auto-generated module) is shown below.
 In this diagram, the instantiation of the Ibex processor and all of the memories and comportable IPs are shown.
 The IO shown at this level are the internal signals between the IP and the pads instantiated at the target top level netlist.
 
-In the block diagram below, the target netlist `chip_earlgrey_nexsysvideo` is shown, including the FPGA pad names created.
-
-![Top Level Block Diagram](chip_earlgrey_nexsysvideo_block_diagram.svg)
-
-In this diagram, pads for clock and reset are shown, as well as pads for JTAG, SPI device, UART, and GPIO.
-In this platform, at the moment, the logic for the JTAG and the SPI device are multiplexed within `chip_earlgrey_nexsysvideo`.
+In the FPGA platform, the logic for the JTAG and the SPI device are multiplexed within `chip_earlgrey_cw310`.
 This is done for ease of programming by the external host.
-In addition, at the moment, the UART pins and GPIO pins are separated out from the multipurpose `MIO` pins.
+In addition, the UART pins and GPIO pins are separated out from the multipurpose `MIO` pins.
```

Comment:
I don't think this is true anymore, since GPIO and UART just go through the pinmux and hence `MIO` pins...

Reply:
Done. Removed this paragraph.